### PR TITLE
Added Gemini Protocol and Gemini Servers

### PR DESCRIPTION
**Pull request by pswilde**
Added the @benob's Gemini library, Gemini Servers group and Nemini, a Gemini Server

### CODE REVIEW

**Comment by zedeus** [comment]
Don't forget https://github.com/ardek66/geminim

**Comment by pswilde** [comment]
> Don't forget https://github.com/ardek66/geminim

yep, added. I've had trouble with that one myself but that's probably just me 😁

File: README.md [hunk 1]
```diff
@@ -596,6 +599,10 @@ Nim provides unique features for seamless and transparent interoperability with
 - [netkit](https://github.com/iocrate/netkit) - Out-of-the-box, stable and secure network facilities and utilities written in pure Nim.
 - [jshttp2](https://juancarlospaco.github.io/nodejs/nodejs/jshttp2) - Async HTTPS 2.0 web server.
 
+### Gemini Servers
+
+- [Geminim](https://github.com/ardek66/geminim) - A Gemini server
+- [Nemini](https://codeberg.org/pswilde/Nemini) - A very simple Gemini server for serving static gemtext files
```
Comment (ringabout) [inline README.md:605]:
```suggestion
- [Geminim](https://github.com/ardek66/geminim) - A Gemini server.
- [Nemini](https://codeberg.org/pswilde/Nemini) - A very simple Gemini server for serving static gemtext files.
```

**Review by ringabout — APPROVED** [review]
Nice work!

**Pull request merged**